.github/workflows/website.yml

```diff
@@ -12,10 +15,16 @@ jobs:
       - name: Set env
         run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
       - name: Render docs and publish
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_API_TOKEN }}
         run: |
           git config --global user.name "Dekorate"
           git config --global user.email "cmoullia-staff@redhat.com"
           sudo apt-get install moreutils
           echo $RELEASE_VERSION
           echo ${{ env.RELEASE_VERSION }}
+          echo ${GITHUB_TOKEN}
+          echo 'about to clone repo'
+          git clone https://aureamunoz:${GITHUB_TOKEN}@github.com/dekorateio/dekorate.git
```

Comment:
I feel that there should be a way to deal with it without having to use our actual user names, email addresses etc.

Using `actions/checkout` could possibly be a more louse approach to it. If there are technical reasons why we need to add a comment as its not obvious.

Reply:
yes, sure! I'm just debugging/testing....I should have created the PR in draft mode, I will